PublicGitArchive/pga-create/README.md

```diff
@@ -74,6 +74,12 @@ The options accepted by `pga-create index` are the following:
 -r, --repositories= input path for the gzipped file with the repository names and identifiers (default: data/repositories.gz)
 ```
 
+To set `MAX_SIVAS_SIZE` field properly, it assumes that the environment variable to set the configuration for the dependency [core-retrieval](https://github.com/src-d/core-retrieval) are the default but for `CONFIG_CLEAN_TEMP_DIR` which you need to set to `true`
```

Comment:
I would call it `SIZE` and explain what that means in the documentation. This will probably change in the future so it's better to change the documentation explaining what it is than the CSV columns.